src/wrapper/clean.rs

```diff
@@ -136,6 +136,23 @@ fn clean_dir_wrapper(path: &Path, dry_run: bool) -> Result<(), String> {
     Ok(())
 }
 
+fn clean_everything(wrapper: &Wrapper, global: bool, dry_run: bool)
+    -> Result<(), String>
+{
+    if global {
+        if let Some(ref cache_dir) = wrapper.ext_settings.cache_dir {
+            try!(clean_dir_wrapper(cache_dir.as_path(), dry_run));
+        }
+        if let Some(ref storage_dir) = wrapper.ext_settings.storage_dir {
+            try!(clean_dir_wrapper(storage_dir.as_path(), dry_run));
+        }
+    } else {
```

Comment:
In non-global case we need to clean the directory of storage dir that is linked to by `.lnk`, not just `.vagga`. (if there is a storage_dir of course).